qa/suites/rgw/sts/tasks/webidentity.yaml

```diff
@@ -0,0 +1,16 @@
+tasks:
+- tox: [ client.0 ]
+- keycloak:
+    client.0:
+- s3tests:
+    client.0:
+      git remote: git://github.com/TRYTOBE8TME/
```

Comment:
all of the git remotes should be pointing to github.com/ceph/ceph for the final version of this PR. You will remove this once the code is ready and tested right?

Reply:
Yes you're correct.